src/daemon/jsonrpc/api.rs

```diff
@@ -333,6 +337,123 @@ impl RpcApi for RpcImpl {
         }))
     }
 
+    fn help(&self, meta: Self::Metadata) -> jsonrpc_core::Result<serde_json::Value> {
+        Ok(json!(
+        {
+            "commands":[
+                {
+                  "name": "getinfo",
+                  "parameters": [
+
+                  ],
```

Comment:
For all empty arrays of parameters, can you put the square brackets on the same line? ie:
```json
"parameters": [],
```